NPE in checkPendingDownloads

### Describe your problem!

Reported via Play console, marked as new in current release:

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3757)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3924)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:103)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2279)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loopOnce (Looper.java:210)
  at android.os.Looper.loop (Looper.java:299)
  at android.app.ActivityThread.main (ActivityThread.java:8291)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:556)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1037)
Caused by: java.lang.NullPointerException: 
  at cgeo.geocaching.downloader.DownloaderUtils.checkPendingDownloads (DownloaderUtils.java:352)
  at cgeo.geocaching.MainActivity.onCreate (MainActivity.java:349)
  at android.app.Activity.performCreate (Activity.java:8178)
  at android.app.Activity.performCreate (Activity.java:8146)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1330)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3730)
```

### How to reproduce?

unclear

### Actual result after these steps?

_No response_

### Expected result after these steps?

_No response_

### Reproducible

Unclear

### c:geo Version

2022.06.06

### System information

_No response_

### Additional Information

Android 11/12, currently 5 occurrences, 2 users with two different Redmi models

--- a/main/src/cgeo/geocaching/downloader/DownloaderUtils.java
+++ b/main/src/cgeo/geocaching/downloader/DownloaderUtils.java
@@ -349,6 +349,10 @@ public class DownloaderUtils {
         if (null != downloadManager) {
             final DownloadManager.Query query = new DownloadManager.Query();
             try (Cursor c = downloadManager.query(query)) {
+                if (c == null) {
+                    Log.w("checkPendingDownloads: querying DownloadManager returned null");
+                    return;
+                }
                 final int columnId = c.getColumnIndex(DownloadManager.COLUMN_ID);
                 final int columnStatus = c.getColumnIndex(DownloadManager.COLUMN_STATUS);
                 final int columnBytesA = c.getColumnIndex(DownloadManager.COLUMN_BYTES_DOWNLOADED_SO_FAR);

